packages/core/src/authMetadata.ts

```diff
@@ -11,18 +11,21 @@ class AuthMetadata implements IAuthMetadata {
 
   privKey: BN;
 
-  constructor(metadata: Metadata, privKey?: BN) {
+  keyType: KeyType;
+
+  constructor(metadata: Metadata, privKey?: BN, keyType?: KeyType) {
```

Comment:
should we define default value on the function signature ?

Reply:
inside constructor i have assigned default value for keytype 
which function do we need default value is it a static function